theme.conf

```diff
@@ -8,3 +8,30 @@ color_main=#dddddd
 color_dimmed=#888888
 color_contrast=#1f1f1f
 manual=false
+user_area_width=450
+power_area_width=370
+offset_slices=5
+padding_slices=0
+spacing_slices=3
+skew_slices=32
+image_padding=2
+spacing_item=2
+padding_item=2
+
+padding_vertical_slices=3
+padding_vertical_slices_login_buttons=2
+offset_vertical_slices_login_buttons=2
+offset_horizontal_slices_login_buttons=0
+
+padding_horizontal_item_session=10
+padding_vertical_item_session=5
+
+padding_horizontal_item_power=10
+padding_vertical_item_power=5
+
+padding_top_item_user=0
+padding_bottom_item_user=-4
+padding_horizontal_item_user=10
+image_padding_item_user=4
+offset_sub_head_item_user=-9
+offset_lower_case_item_user=7
```

Comment:
`theme.conf` is only for required settings, however in `SizeScheme.qml` these settings are on higher levels, which means that they should fall back to Layer 1 values. Defining these values in `theme.conf` prevents falling back to Layer 1.

There are multiple ways to solve this.

1. These values can be moved to Layer 1 in `SizeScheme.qml`. However, global values may be affected not in a good way, so it needs to be checked.
2. Some new options may be introduced, like additional paddings or multipliers.
3. These values also can be removed from `theme.conf` and hardcoded. It may be not a good fit for layout values scheme, but it should work as a temporary solution at least.